cura/CuraPackageManager.py

```diff
@@ -245,22 +245,17 @@ def removePackage(self, package_id: str) -> None:
 
     # Removes everything associated with the given package ID.
     def _purgePackage(self, package_id: str) -> None:
-        # Get all folders that need to be checked for installed packages, including:
-        #  - materials
-        #  - qualities
-        #  - plugins
-        from cura.CuraApplication import CuraApplication
-        dirs_to_check = [
-            Resources.getStoragePath(CuraApplication.ResourceTypes.MaterialInstanceContainer),
-            Resources.getStoragePath(CuraApplication.ResourceTypes.QualityInstanceContainer),
-            os.path.join(os.path.abspath(Resources.getDataStoragePath()), "plugins"),
-        ]
-
-        for root_dir in dirs_to_check:
-            package_dir = os.path.join(root_dir, package_id)
-            if os.path.exists(package_dir):
-                Logger.log("i", "Removing '%s' for package [%s]", package_dir, package_id)
+        # Iterate through all directories in the data storage directory and look for sub-directories that belong to
+        # the package we need to remove, that is the sub-dirs with the package_id as names, and remove all those dirs.
+        data_storage_dir = os.path.abspath(Resources.getDataStoragePath())
+
+        for root, dir_names, _ in os.walk(data_storage_dir):
+            for dir_name in dir_names:
+                package_dir = os.path.join(root, dir_name, package_id)
+                if os.path.exists(package_dir):
+                    Logger.log("i", "Removing '%s' for package [%s]", package_dir, package_id)
                 shutil.rmtree(package_dir)
```

Comment:
Should this be within the previous `if` statement?

Reply:
Fixed in 62aeb6c